addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -154,19 +156,25 @@
             <t t-set="tickets" t-value="event.event_ticket_ids.filtered(lambda ticket: not ticket.is_expired)"/>
             <!-- If some tickets expired and there is only one type left, we keep the same layout -->
             <t t-if="len(event.event_ticket_ids) &gt; 1 or tickets.description">
-                <div class="d-flex justify-content-between align-items-center py-2 pl-3 pr-2 border-bottom">
-                    <div id="price-range" class="pr-3 d-none"/>
-                    <span t-if="not event.event_registrations_open" class="text-danger">
+                <div class="d-flex align-items-center py-2 pl-3 pr-2 border-bottom">
+                    <span id="tickets-title" class="btn py-2 o_wevent_registration_title text-left">Tickets</span>
+                    <div id="price-range" class="d-none"/>
+                    <div t-if="request.env.user.has_group('event.group_event_manager')" class="o_not_editable ml-2" role="link">
+                        <a t-attf-href="/web#id=#{event.id}&amp;view_type=form&amp;model=event.event">
+                            <i class="fa fa-gear mr-2" role="img" aria-label="Configure" title="Configure event tickets"/><em>Configure Tickets</em>
+                        </a>
+                    </div>
+                    <span t-if="not event.event_registrations_open" class="text-danger o_toggle-d-none">
                         <i class="fa fa-ban mr-2"/>Sold Out
                     </span>
-                    <div t-if="event.is_participating" id="registered" class="ml-auto pr-3 d-none">
+                    <div t-if="event.is_participating" id="registered" class="ml-auto pr-3 d-none o_toggle-d-none">
```

Comment:
Doesn't seem to work the way you want it to, stays displayed when expanded:
![image](https://user-images.githubusercontent.com/41282536/113983293-098a6a00-984a-11eb-8990-fa5f1c23ae3f.png)

Old version doesn't show it when expanded:
![image](https://user-images.githubusercontent.com/41282536/113983344-17d88600-984a-11eb-82c7-e3995b18dc67.png)

Or maybe that's something we want to show all the time now?
But then why the "o_toggle-d-none" thing?

EDIT: just noticed the cross to close is replaced by a "tickets" button on my first screenshot??
Seems to happen if I expand the button too quickly (old version seems to have that issue as well).
Really strange.